frontend/server/configs.ts

```diff
@@ -91,8 +91,8 @@ export function loadConfigs(argv: string[], env: ProcessEnv): UIConfigs {
     ARGO_ARCHIVE_ARTIFACTORY = 'minio',
     /** Bucket to retrive logs from */
     ARGO_ARCHIVE_BUCKETNAME = 'mlpipeline',
-    /** Prefix to logs. */
-    ARGO_ARCHIVE_PREFIX = 'logs',
+    /** This should match the keyFormat specified in the Argo workflow-controller-configmap */
+    ARGO_KEYFORMAT = 'artifacts/{{workflow.name}}/{{workflow.creationTimestamp.Y}}/{{workflow.creationTimestamp.m}}/{{workflow.creationTimestamp.d}}/{{pod.name}}',
```

Comment:
It will be great to add a link to the source of this log path in the comment. Whenever there is a location change in the future, we can still find out by looking at the source of truth.

Reply:
Copy that! Do you prefer referencing it by:

1. Master branch link:
https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/third-party/argo/base/workflow-controller-configmap-patch.yaml#L28
3. Permalink: https://github.com/kubeflow/pipelines/blob/7b7918ebf8c30e6ceec99283ef20dbc02fdf6a42/manifests/kustomize/third-party/argo/base/workflow-controller-configmap-patch.yaml#L28
4. The file path relative to `config.ts`:
../../manifests/kustomize/third-party/argo/base/workflow-controller-configmap-patch.yaml

Reply:
Thanks for the question! I am fine with your preference for any of that above, I would `link to the premalink` for the stability of content we are referring.